src/integrationTest/java/uk/gov/hmcts/reform/orgrolemapping/config/SwaggerPublisher.java

```diff
@@ -20,26 +20,31 @@
  * Built-in feature which saves service's swagger specs in temporary directory.
  * Each travis run on master should automatically save and upload (if updated) documentation.
  */
-@SpringJUnitWebConfig
-@SpringBootTest
-@AutoConfigureMockMvc
-class SwaggerPublisher {
+public class SwaggerPublisher extends BaseTest {
+
+    private MockMvc mockMvc;
 
-    @Autowired
-    private transient MockMvc mvc;
+    @Inject
+    private WebApplicationContext wac;
+
+    @Before
+    public void setUp() {
+        mockMvc = MockMvcBuilders.webAppContextSetup(wac).build();
+    }
 
     @DisplayName("Generate swagger documentation")
     @Test
     @Disabled
```

Comment:
Should it realy be disabled?

Reply:
Disabled removed

Reply:
Resolved in e9411a965550fa4fb436d81c7730e64acf15fbcd